binding/form_mapping.go

```diff
@@ -376,6 +380,25 @@ func head(str, sep string) (head string, tail string) {
 	return str[:idx], str[idx+len(sep):]
 }
 
+func split(vals []string, field reflect.StructField) []string {
+	if cfTag := field.Tag.Get("collection_format"); cfTag != "" {
+		sep := "multi"
```

Comment:
It will stay as `sep = "multi"` when invalid separator is defined e.g. `collection_format:"foo"`
Should it fail / warn instead silently doing fallback?
Should also add test case for it.